Source/WebCore/rendering/RenderBox.cpp

```diff
@@ -5630,4 +5648,14 @@ std::optional<LayoutUnit> RenderBox::explicitIntrinsicInnerHeight() const
     return std::optional<LayoutUnit> { height->value() };
 }
 
+RenderBlockFlow* RenderBox::blockFormattingContextRoot() const
+{
+    // This method should probably not be called on the initial containing block
+    ASSERT(!is<HTMLHtmlElement>(element()));
+    auto cb = containingBlock();
```

Comment:
please try not to abbreviate unless absolutely necessary.

Reply:
Fixed